module/x/gravity/client/cli/query.go

```diff
@@ -197,6 +198,29 @@ func CmdGetPendingSendToEth() *cobra.Command {
 	return cmd
 }
 
+func GetCmdPendingIbcAutoForwards() *cobra.Command {
+	//nolint: exhaustivestruct
```

Comment:
Oops, forgot to add a parameter for results limit.

The command should look like `gravity query pending-ibc-auto-forwards [limit]`